src/runtime/propensity-server.js

```diff
@@ -56,8 +59,8 @@ export class PropensityServer {
     this.logSwgEventsExperiment_ = isExperimentOn(win,
         ExperimentFlags.LOG_SWG_TO_PROPENSITY);
 
-    /** @private {!boolean} */
-    this.logSwgEventsConfig_ = false;
+    /** @private {!function():?boolean} */
+    this.logSwgEventsCallback_ = logSwgEventsCallback || (() => false);
```

Comment:
Do you want a function that returns true or false or config object?
Why not get entire config returned in the callback and then you can access that exact field

Reply:
I guess I'm not clear on what propensity can rely on.  It can rely on a properly formed config object but not a depsdef?

Reply:
I think using a boolean keeps Propensity's dependence on SwG as a whole minimal.